umbraco/Umbraco-CMS
# V14: List view reference in ContentType

# Notes 
- Refactored `IsContainer` from ContentType to `ListView` which is a Guid, which is a direct reference to a datatype.
- Refactored the viewmodels to now contain `Collection` which is a `ReferenceByIdModel`.
- Implemented migration with logic to convert the bool value, to a given ListView (or null if not a container)

# How to test
- Test the migration on both SqlServer & SQLite
-- Create a v13 database, with lots of different DocumentTypes & MediaTypes
-- Create some without and with list views, and some with custom list views.
-- Switch to this branch, and migrate the v13 database, assert that the migration works as expected, and the correct list view keys are referenced.


- Test the output of the swagger
- When querying a DocumentType/MediaType/MemberType, assert that the swagger is correct
- use the ``get` endpoints for each type, and assert it has the correct `Collection` value (null for no list view, correct key if it has a list view)
